src/Spectre.AngularClient/tslint.json

```diff
@@ -34,7 +34,6 @@
     "no-bitwise": false,
     "no-console": [
       true,
-      "debug",
```

Comment:
No, I was looking for something to switch off console in prod but it is hard to find